ide/o.n.swing.dirchooser/nbproject/project.properties

```diff
@@ -14,5 +14,6 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-javac.source=1.8
+javac.source=11
+javac.target=11
```

Comment:
@neilcsmith-net the tests did also not warn about the missing JDK requirement field of the manifest here, even though this is a regular module.

Reply:
@mbien unfortunately as expected then!  Need to fix both the manifest generation and the test.  Do you want to fix the manifest generation as a separate commit in this PR?

Reply:
continuing discussion over there https://github.com/apache/netbeans/pull/7180#discussion_r1537814701